iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/GeneratorModification.java

```diff
@@ -106,6 +120,7 @@ public static class Modifs {
         private Double targetQ;
         private Boolean voltageRegulatorOn;
         private Boolean connected;
+        private Boolean doNotRestrictTargetPAndDoNotConnect;
```

Comment:
A simple `boolean` (with a default value set to `false`) can be used for this attribute.
It hasn't the same behavior as `voltageRegulatorOn` and `connected` which can have 3 states (for instance with `connected`: `null` = do nothing, `true` = connect the generator, `false` = disconnect the generator).

Reply:
fixed